src/hardware/setup.rs

```diff
@@ -665,6 +665,46 @@ pub fn setup(
             )
         };
 
+        // Note(unsafe): The MMC_TX/RX_INTERRUPT_MASK registers incorrectly mark LPITRCIM as
+        // read-only, so Svd2rust doens't generate bindings to modify them. Instead, as a
+        // workaround, we manually manipulate the bits.
+        unsafe {
+            let eth_mac = &*hal::stm32::ETHERNET_MAC::ptr();
+
+            // Mask away Ethernet MAC MMC RX/TX interrupts. These are statistics counter interrupts
+            // and are enabled by default. They are intended to provide ethernet MAC-related
+            // statistics, which we are not using.
+            eth_mac.mmc_rx_interrupt_mask.modify(|_, w| {
+                w.rxlpiuscim()
+                    .set_bit()
+                    .rxucgpim()
+                    .set_bit()
+                    .rxalgnerpim()
+                    .set_bit()
+                    .rxcrcerpim()
+                    .set_bit()
+            });
+
+            eth_mac.mmc_tx_interrupt_mask.modify(|_, w| {
+                w.txlpiuscim()
+                    .set_bit()
+                    .txgpktim()
+                    .set_bit()
+                    .txmcolgpim()
+                    .set_bit()
+                    .txscolgpim()
+                    .set_bit()
+            });
+
+            let mmc_tx_interrupt_mask: *mut u32 =
+                &eth_mac.mmc_tx_interrupt_mask as *const _ as *mut u32;
+            *mmc_tx_interrupt_mask |= 1u32 << 27;
```

Comment:
Can't we just `unsafe { eth_mac.mmc_tx_interrupt_mask.modify(|_, w| w.bits(0xffff_ffff)) }`?
Less code to maintain until the PAC is fixed and less fragile.

Reply:
Other bits in this register are marked as "Reserved, must be kept at reset value"

Reply:
Then `w.bits(r.bits() | (1u32 << 27))`.

Reply:
Much nicer suggestion